sonar-python-plugin/src/main/java/org/sonar/plugins/python/caching/Caching.java

```diff
@@ -43,6 +43,7 @@ public class Caching {
   public static final String IMPORTS_MAP_CACHE_KEY_PREFIX = "python:imports:";
   public static final String PROJECT_SYMBOL_TABLE_CACHE_KEY_PREFIX = "python:descriptors:";
   public static final String PROJECT_FILES_KEY = "python:files";
+  public static final String CONTENT_HASHES_KEY = "python:content_hashes";
```

Comment:
_minor_, but the key would look a bit nicer with the colon in the end:
```suggestion
  public static final String CONTENT_HASHES_KEY = "python:content_hashes:";
```